docs_espressif/en/additionalfeatures/coverage.rst

```diff
@@ -0,0 +1,70 @@
+ESP-IDF Coverage
+===============================
+
+Source code coverage is data indicating the count and frequency of every program execution path that has been taken within a program runtime. `Gcov <https://en.wikipedia.org/wiki/Gcov>`_ is a GCC tool that, when used in concert with the compiler, can generate log files indicating the execution count of each line of a source code.
+
+Your ESP-IDF project should be configured to generate ``gcda/gcno`` coverage files using ``gcov``. Please read `GCOV Code Coverage <https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-guides/app_trace.html#gcov-source-code-coverage>`_ to learn more about code coverage with GCOV in ESP-IDF projects.
+
+You can use the **ESP-IDF: Configure Project SDKConfig for Coverage** to set required configuration in the SDK Configuration Editor. 
+
+Code coverage example
+-------------------------------
+
+Let's use the ESP-IDF `GCOV Example <https://github.com/espressif/esp-idf/tree/master/examples/system/gcov>`_ for this tutorial.
+
+Click menu **View** -> **Command Palette...** and type **ESP-IDF: Show Examples Projects** and choose ``Use Current ESP-IDF (/path/to/esp-idf)``. If the user doesn't see the option, please review the setup in Installation <installation>.
+
+A window will be open with a list a projects, go the **system** section and choose the `gcov`. You will see a **Create Project Using Example gcov** button in the top and a description of the project below. Click **Create Project Using Example GCOV** button.
+
+.. image:: ../../../media/tutorials/coverage/gcov_example.png
+
+Now select a container directory where to copy the example project. For example, if the user choose ``/Users/myUser/someFolder`` the resulting folder will be ``/Users/myUser/someFolder/gcov``. This new project directory will be created and opened in Visual Studio Code.
```

Comment:
For consistency in addressing the reader, I would suggest addressing him directly and not use the `user` word here